services/update.py

```diff
@@ -13,9 +10,6 @@
 # This has to be imported after the settings so statsd knows where to log to.
 from django_statsd.clients import statsd
 
-import MySQLdb as mysql
-import sqlalchemy.pool as pool
```

Comment:
just looked this up, this was old stuff and we are using `services.utils:mypool` (pretty sure that variable name got copied from a tutorial back then) which is shared across `services`.

Reply:
... is there something wrong/I need to do here?  Or just saying?

Reply:
ah, just me blabbering out stuff… Nothing to change, all good - that's basically what that comment tries to say :)